runtime/calamari/src/lib.rs

```diff
@@ -497,20 +500,19 @@ impl pallet_membership::Config<TechnicalMembershipInstance> for Runtime {
 parameter_types! {
 	pub const ProposalBond: Permill = Permill::from_percent(1);
 	pub const ProposalBondMinimum: Balance = 50 * KMA;
+	pub const ProposalBondMaximum: Balance = 1000 * KMA;
```

Comment:
https://github.com/paritytech/substrate/pull/10689

Reply:
Arbitrary choice.
The idea for the cap seems to be to not discourage people from proposing really big spends.
1k KMA (~7 .5USD) is probably too small though.
Kusama's cap is 1k KSM
Karura's cap is 25 dollars
KILT and Interlay don't even set it.

IMO we can change it to like 100 USD